### PR TITLE
Fat 4773 c368005

### PR DESCRIPTION
**Pull request by TetianaParanich**
***
https://issues.folio.org/browse/FAT-4773
***
https://foliotest.testrail.io/index.php?/cases/view/368005
***
https://jenkins-aws.indexdata.com/job/Testing/job/Cypress_tests/1637/allure/
***

### CODE REVIEW

File: cypress/support/fragments/data_import/mapping_profiles/newFieldMappingProfile.js [hunk 1]
```diff
@@ -401,6 +403,23 @@ export default {
...
+    const noteFieldName = 'profile.mappingDetails.mappingFields[29].repeatableFieldAction';
+    const selectName = 'profile.mappingDetails.mappingFields[29].subfields[0].fields[2].booleanFieldAction';
+
+    cy.do([
+      Select({ name:noteFieldName }).focus(),
+      Select({ name:noteFieldName }).choose(actions.addTheseToExisting),
+      Button('Add check in / check out note').click(),
+      loanAndAvailabilityAccordion.find(TextField('Note type')).fillIn(noteType),
+      loanAndAvailabilityAccordion.find(TextField('Note')).fillIn(note),
+      Select({ name:selectName }).focus(),
```
Comment (IhorBohdan) [inline cypress/support/fragments/data_import/mapping_profiles/newFieldMappingProfile.js:416]: move to const

File: cypress/support/fragments/inventory/itemRecordView.js [hunk 2]
```diff
@@ -76,12 +77,27 @@ export default {
...
 
-  checkItemNote:(note) => {
-    cy.expect(Accordion('Item notes').find(KeyValue('Note')).has({ value: note }));
+  checkCheckInNote:(note, staffValue = 'Yes') => {
+    cy.expect(loanAccordion.find(KeyValue('Check in note')).has({ value: note }));
+    cy.expect(HTML(staffValue).exists());
+  },
+
+  checkCheckOutNote:(note, staffValue = 'Yes') => {
+    cy.expect(loanAccordion.find(KeyValue('Check out note')).has({ value: note }));
```
Comment (IhorBohdan) [inline cypress/support/fragments/inventory/itemRecordView.js:91]: move to const

**Review by IhorBohdan — APPROVED** [review]

File: cypress/support/fragments/inventory/itemRecordView.js (hunk 2 above)
Comment (TetianaParanich) [reply cypress/support/fragments/inventory/itemRecordView.js:91]: Done

File: cypress/support/fragments/data_import/mapping_profiles/newFieldMappingProfile.js (hunk 1 above)
Comment (TetianaParanich) [reply cypress/support/fragments/data_import/mapping_profiles/newFieldMappingProfile.js:416]: Done

File: cypress/support/fragments/data_import/mapping_profiles/newFieldMappingProfile.js [hunk 3]
```diff
@@ -401,6 +403,23 @@ export default {
     waitLoading();
   },
 
+  addCheckInCheckOutNote:(noteType, note, staffOnly) => {
+    const noteFieldName = 'profile.mappingDetails.mappingFields[29].repeatableFieldAction';
+    const selectName = 'profile.mappingDetails.mappingFields[29].subfields[0].fields[2].booleanFieldAction';
+
+    cy.do([
+      Select({ name:noteFieldName }).focus(),
```
Comment (penfriend) [inline cypress/support/fragments/data_import/mapping_profiles/newFieldMappingProfile.js:411]: move to const

**Review by OllIsanbaev — APPROVED** [review]

**Pull request merged**